### PR TITLE
test: Add some checks about OS input on the creation and import disk dialog

### PR DESCRIPTION
**Pull request by yunmingyang**

### CODE REVIEW

**Review by martinpitt — DISMISSED** [review]
Thanks @yunmingyang : I converted this to Draft, as the new test still fails. Also, can you please add "Closes #14644" to the topmost commit message on next push? Cheers!

**Comment by yunmingyang** [comment]
@martinpitt Thanks! I will add it on next push. Then I checked the fails, and found that 'Ctrl + a' in the set_input_text did not take effect. I guess maybe this is related with the version of chromium, so what is the version of chromium in our auto triggered tests? If I get the version I think I can try to reproduce in my laptop.

**Comment by martinpitt** [comment]
@yunmingyang : The current [cockpit/tasks container](https://hub.docker.com/r/cockpit/tasks/tags/) has chromium 81. It's a bit old by now, the last container refresh was 3 months ago. If that is significant, we can certainly bump it again.

File: test/verify/check-machines [hunk 1]
```diff
@@ -2471,6 +2506,11 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
 
             return self
 
+        def checkOsInput(self, dialog):
```
Comment (KKoukiou) [inline test/verify/check-machines:2513]: Can you please add a Test suffix to this function?

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Comment by KKoukiou** [comment]
@yunmingyang unfortunately this is still failing https://logs.cockpit-project.org/logs/pull-14644-20201001-031320-f8b7ffec-fedora-32/log.html#235

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**